tests/Mail/MailMailerTest.php

```diff
@@ -91,9 +92,9 @@ public function testMailerCanQueueMessagesToItselfOnAnotherQueue()
     public function testMailerCanQueueMessagesToItselfWithSerializedClosures()
     {
         list($view, $swift) = $this->getMocks();
-        $mailer = new Illuminate\Mail\Mailer($view, $swift);
+        $mailer = new MailerWithFasterSerializer($view, $swift);
```

Comment:
The point of this PR is precisely to avoid running the default AstAnalyzer (which is incredibly slow even for an empty closure) while touching Mailer the less possible.